sdcm/nemesis.py

```diff
@@ -1022,7 +1022,19 @@ def _choose_file_for_destroy(self, ks_cfs):
 
     @decorate_with_context(ignore_ycsb_connection_refused)
     def _destroy_data_and_restart_scylla(self):
-
+        def find_five_files_to_destroy():
```

Comment:
there no sleep in the loop here, so it might run in quick secession and all might fail to retrieve a file 
I would think some wait is needed between the trials, and once there's enough files, this loop can stop.

Reply:
`_choose_file_for_destroy` already has a retry, so there's that.
Also, I don't see why you would need to wait between the trials. It's not like more are going to be created since scylla is down at this point.

Reply:
We do this deletions when the scylla node is already stopped, so there is no reason to wait.